lib/modules/apostrophe-login/index.js

```diff
@@ -455,7 +455,7 @@ module.exports = {
     }
 
     self.getThrottleLoginErr = function (req, minutes) {
-      return req.__ns_n(
```

Comment:
We're not really using `__ns_n` anywhere else. looked at the i18n docs for the related method and I don't know if our string here would work. We could change the arguments passed to the original method, though I wasn't clear on that using our wrapper method and there weren't relevant other usages to use as a guide. Ultimately using the simpler method gets this message the same place it woudl have got to before.